tests/react/abortable.test.tsx

```diff
@@ -42,21 +43,21 @@ describe('abortable atom test', () => {
     )
 
     await findByText('loading')
+
     resolve.splice(0).forEach((fn) => fn())
-    await findByText('count: 0')
+    expect(await findByText('count: 0')).toBeTruthy()
```

Comment:
I mean, just leave it unchanged in this PR.
```suggestion
    await findByText('count: 0')
```